### PR TITLE
Angela Guardia

### PR DESCRIPTION
**Pull request by guardiac**

### CODE REVIEW

File: lib/item.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Item
+  attr_reader :name,
+              :price
+
+  def initialize(info)
+    @name = info[:name]
+    @price = info[:price].slice(1, 4).to_f
```
Comment (mikedao) [inline lib/item.rb:7]: This is good, another alternative is you can do `info[:price][1..-1]`

File: lib/market.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+class Market
+  attr_reader :name,
+              :vendors,
+              :date
+
+  def initialize(name)
+    @name = name
+    @vendors = []
+    @date = Date.today.strftime("%d/%m/%Y")
```
Comment (mikedao) [inline lib/market.rb:11]: I would probably have made the date a method that used strftime, so you are holding more data.

File: lib/market.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  end
+
+  def total_inventory
+    available_items.reduce(Hash.new { |h,k| h[k] = {} }) do |result, item|
+      result[item] = {quantity: item_stock(item), vendors: vendors_that_sell(item)}
+      result
+    end
+  end
+
+  def available_items
```
Comment (mikedao) [inline lib/market.rb:33]: I really like this.

File: lib/item.rb (hunk 1 above)
Comment (guardiac) [reply lib/item.rb:7]: Thanks for that suggestion Mike! That way it could work for strings of different lengths, I like it.

**Pull request closed without merging**